src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -116,7 +117,8 @@ def load_arguments(self, _):
     with self.argument_context('appconfig kv export', arg_group='AppConfig') as c:
         c.argument('dest_name', help='The name of the destination App Configuration.')
         c.argument('dest_connection_string', validator=validate_connection_string, help="Combination of access key and endpoint of the destination store.")
-        c.argument('dest_label', help="Exported KVs will be labeled with this destination label.")
+        c.argument('dest_label', help="Exported KVs will be labeled with this destination label. If neither --dest-label nor --preserve-labels is specified, will assign null label.")
+        c.argument('preserve_labels', arg_type=get_three_state_flag(), help="Flag to preserve labels from source AppConfig. This argument should NOT be specified along with --dest-label.")
```

Comment:
why does it preserve labels from source AppConfig? Can you explain --preserve-lables in detail to me?

Reply:
All AppConfig keys are associated with a label. When we import keys from another AppConfig (source), users can specify what label they want to assign to each imported key in the dest AppConfig. --preserve-labels is a flag that is used to import keys along with their existing labels in the source AppConfig.

---
In reply to: [365274049](https://github.com/Azure/azure-cli/pull/11756#discussion_r365274049) [](ancestors = 365274049)